src/pykeen/datasets/ogb.py

```diff
@@ -104,6 +104,41 @@ class OGBBioKG(OGBLoader):
 
     name = 'ogbl-biokg'
 
+    def _load(self) -> None:
+        try:
+            from ogb.linkproppred import LinkPropPredDataset
+        except ImportError as e:
+            raise ModuleNotFoundError(
+                f'Need to `pip install ogb` to use pykeen.datasets.{self.__class__.__name__}.',
+            ) from e
+
+        dataset = LinkPropPredDataset(name=self.name, root=self.cache_root)
+        edge_split = dataset.get_edge_split()
+        self._training = self._make_tf(edge_split["train"])
+        assert self._training is not None  # makes mypy hapy
+        self._testing = self._make_tf(
+            edge_split["test"],
+            entity_to_id=self._training.entity_to_id,
+            relation_to_id=self._training.relation_to_id,
+        )
+        self._validation = self._make_tf(
+            edge_split["valid"],
+            entity_to_id=self._training.entity_to_id,
+            relation_to_id=self._training.relation_to_id,
+        )
+
+    def _make_tf(self, x, entity_to_id=None, relation_to_id=None):
+        head_triples = np.array(list(map(lambda t,e: t+':'+str(e), x['head_type'], x['head'])), dtype=np.str)
```

Comment:
I think there is an issue with the implementation here - `x['head_type']` and `x['head']` should be zipped together before passed as the second positional argument to `map()`.

Additionally, the use of `list(map(lambda ...` is quite overcomplicated compared to a simple list comprehension. Another improvement is to use an f-string to remove the need for string casting and concatenation

```suggestion
        head_triples = np.array([f'{head_type}:{head}' for head_type, head in zip(x['head_type'], x['head']], dtype=np.str)

Reply:
same thing for the following line

Reply:
Thanks for the review, and good idea. Made these changes, see newest commit.